ansible/roles/letsencrypt/tasks/letsencrypt_renew.yml

```diff
@@ -4,5 +4,5 @@
     name: "Renew ooni-backend letsencrypt certificates"
     user: "root"
     cron_file: "letsencrypt_renew_certs-{{ app_env }}"
-    special_time: "monthly"
+    special_time: "weekly"
```

Comment:
This cronjob will check if renewal is necessary and will try to renew the required certificates if it's needed (i.e certificate's expiration date is approaching).
During the renewal checks there is no service interruption.

When the certificates are due for renewal, certbot (letsencrypt's component) will start the renewal process by bringing down the appropriate container, renew the certificates, copy the updated certificates in oonibackend's certificate storage and then start the appropriate container for oonibackend to read the updated certificates.

Currently the deployed oonibackend service is set to check for certificate renewals monthly, we can decrease the renewal time period to daily, though setting the cronjob to run hourly, weekly or monthly has nothing to do with #80 (if this is the bug mentioned by not working as expected).

The oonibackend services need to be taken offline in order for oonibackend to read the newly generated certificates (see: https://github.com/TheTorProject/ooni-backend/issues/90).